src/Makefile

```diff
@@ -694,6 +706,11 @@ rshapes.o : rshapes.c raylib.h rlgl.h
 rtextures.o : rtextures.c raylib.h rlgl.h utils.h
 	$(CC) -c $< $(CFLAGS) $(INCLUDE_PATHS)
 
+RGFW.o: 
+	cp external/RGFW.h RGFW.c
```

Comment:
Does `cp` work on Windows? Also MSVC needs to be considered, a Visual Studio Project is included with raylib.

Reply:
I believe cp should work with make because make is a gnu tool. 

Although I'll double check that later.

Reply:
> Does `cp` work on Windows? Also MSVC needs to be considered, a Visual Studio Project is included with raylib.

Does the makefile even support msvc?

Reply:
A number of things are being inter-confused here.  First, [make](https://en.wikipedia.org/wiki/Make_(software)) originated in 1976 on Unix and various enhancements and variations have been made since.  

The Visual Studio Build Tools and Visual Studio supply the tool [nmake](https://learn.microsoft.com/en-us/cpp/build/reference/nmake-reference?view=msvc-170), which also preceded Linux (but maybe not GNU).

It is the case that `nmake` accepts makefiles, although there might need to be differences in some setting macros and accommodation of differences with respect to other `make` forks.

Whether the makefile you are speaking of is specific to use of gcc and has any GNU/Linux-tool dependencies, like the Unix `cp` command, it all depends.

Reply:
Thank you for explaining